src/__swaps__/screens/Swap/hooks/useSwapInputsController.ts

```diff
@@ -72,6 +83,7 @@ export function useSwapInputsController({
   quote,
   sliderXPosition,
   slippage,
+  outputQuotesAreDisabled,
```

Comment:
this param no longer needed and the corresponding changes in `swap-provider` also not needed